spec/models/group_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  before(:all) do
+    @category1 = Group.create(name: 'Category 1', author_id: 1)
+    @category1.icon.attach(io: File.open(Rails.root.join('spec', 'images', 'test.jpg')), filename: 'test.jpg',
+                           content_type: 'image/jpg')
+  end
+
+  it 'should have a name' do
+    expect(@category1.name).to eq('Category 1')
+    expect(@category1).to be_valid
+  end
+
+  it 'should be invalid wihtout a name' do
+    @category1.name = nil
+    expect(@category1).to be_invalid
+  end
```

Comment:
- Kindly fix the failing text as seen below 😉.
![Screenshot from 2022-08-12 07-09-28](https://user-images.githubusercontent.com/44624138/184295668-33d3d5f3-3a58-4bb8-9ddf-db7b420e502c.png)

Reply:
* This issue hasnt been resolved kindly work on the failing test